src/_backend_agg_wrapper.cpp

```diff
@@ -95,7 +95,7 @@ int PyBufferRegion_get_buffer(PyBufferRegion *self, Py_buffer *buf, int flags)
     Py_INCREF(self);
     buf->obj = (PyObject *)self;
     buf->buf = self->x->get_data();
-    buf->len = self->x->get_width() * self->x->get_height() * 4;
+    buf->len = (long) self->x->get_width() * (long) self->x->get_height() * 4;
```

Comment:
We can use the final type here, I think?
```suggestion
    buf->len = (Py_ssize_t)self->x->get_width() * (Py_ssize_t)self->x->get_height() * 4;
```